test/AWS.Deploy.CLI.IntegrationTests/SaveCdkDeploymentProject/RedeploymentTests.cs

```diff
@@ -0,0 +1,175 @@
+            var incompatibleDeploymentProjectPath = Path.Combine(tempDirectoryPath, "DeploymentProjects", "IncompatibleCdkApp");
+
+            // perform inital deployment using ECS Fargate recipe
+            await PerformInitialDeployment(projectPath);
+
+            // Create a compatible CDK deployment project using the ECS recipe
+            await Utilities.CreateCDKDeploymentProjectWithRecipeName(projectPath, "Custom ECS Fargate Recipe", "1", compatibleDeploymentProjectPath);
+
+            // Create an incompatible CDK deployment project using the App runner recipe
+            await Utilities.CreateCDKDeploymentProjectWithRecipeName(projectPath, "Custom App Runner Recipe", "2", incompatibleDeploymentProjectPath);
+
+            // attempt re-deployment using incompatible CDK project
+            var deployArgs = new[] { "deploy", "--project-path", projectPath, "--deployment-project", incompatibleDeploymentProjectPath, "--stack-name", _stackName, "--diagnostics" };
+            var returnCode = await _app.Run(deployArgs);
+            Assert.Equal(CommandReturnCodes.USER_ERROR, returnCode);
+
+            // attempt re-deployment using compatible CDK project
+            await _interactiveService.StdInWriter.WriteAsync(Environment.NewLine); // Select default option settings
+            await _interactiveService.StdInWriter.FlushAsync();
+            deployArgs = new[] { "deploy", "--project-path", projectPath, "--deployment-project", compatibleDeploymentProjectPath, "--stack-name", _stackName, "--diagnostics" };
+            returnCode = await _app.Run(deployArgs);
+            Assert.Equal(CommandReturnCodes.SUCCESS, returnCode);
+            Assert.Equal(StackStatus.UPDATE_COMPLETE, await _cloudFormationHelper.GetStackStatus(_stackName));
+            var cluster = await _ecsHelper.GetCluster(_stackName);
+            Assert.Equal("ACTIVE", cluster.Status);
+
+            // Delete stack
+            await DeleteStack();
+        }
+
+        private async Task PerformInitialDeployment(string projectPath)
+        {
+            // Arrange input for deploy
+            await _interactiveService.StdInWriter.WriteLineAsync("1"); // Select ECS Fargate recommendation
+            await _interactiveService.StdInWriter.WriteAsync(Environment.NewLine); // Select default option settings
+            await _interactiveService.StdInWriter.FlushAsync();
+
+            // Deploy
+            var deployArgs = new[] { "deploy", "--project-path", projectPath, "--stack-name", _stackName, "--diagnostics" };
+            await _app.Run(deployArgs);
+
+            // Verify application is deployed and running
+            Assert.Equal(StackStatus.CREATE_COMPLETE, await _cloudFormationHelper.GetStackStatus(_stackName));
+
+            var cluster = await _ecsHelper.GetCluster(_stackName);
+            Assert.Equal("ACTIVE", cluster.Status);
```

Comment:
Can use ECS constants for ACTIVE?  Also L93.

https://docs.aws.amazon.com/sdkfornet/v3/apidocs/items/ECS/TTaskDefinitionStatus.html

Reply:
Fixed